src/view/commands/submit_request.rs

```diff
@@ -30,20 +31,23 @@ impl BasicRequestExecutor<CrosstermCliWriter, CrosstermCliWriter, CrosstermCliWr
         if view_options.print_body_only {
             BasicRequestExecutor {
                 request,
+                progress_draw_target: ProgressDrawTarget::stderr(),
```

Comment:
Instead insert this ProgressDrawTarget in struct, use just a booleand field, like "show_loading". And check this value for hide or show the loading.

The CommandExecutor dependencies should be decoupled as possible of crate we are using to print stuff. If we need to change from Indicatif or Crossterm for another crate, the change doesn't be so painful.